### DISCUSSION TITLE
Is there a way to use semantic HTML tags with the Box JSX pattern?

### DISCUSSION THREAD

**Discussion by NateWritesCode** [chakra-ui/panda#2603] [Q&A] [answered]
Hello,

I was wondering if there is a way to use semantic HTML tags with the Box JSX pattern.

When I used to use Chakra, I believe I was able to use the following...

```
<Box 
    as="aside"
>
  This is the box content
</Box
```

After perusing the docs, I wasn't able to find anything similar in Panda. 

Thanks! And please let me know if you have any questions for me.

Nathan

**Comment by kiruushaaa** [comment, marked as answer]
actually `Box` without any props is a simple `<div class>` so you can use `<styled.aside>` to achieve that

**Reply by NateWritesCode** [reply]
Thanks so much!
